content/blog/openemr-achieves-onc-certification-with-groundbreaking-release-70.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: OpenEMR Achieves ONC 2015 Cures Update Certification with Groundbreaking Release 7.0
+title: OpenEMR Achieves ONC Certification with Groundbreaking Release 7.0
 author:
   name: OpenEMR
   twitter: openemr
 date: '2022-07-19'
-slug: openemr-achieves-onc-2015-cures-update-certification-with-groundbreaking-release-70
+slug: openemr-achieves-onc-certification-with-groundbreaking-release-70
```

Comment:
hi @robertdown ,
Is there a way to support 2 slugs so that links (using old slug) that go to this blog article don't break?